frontend/src/liboperator/operator/devices.go

```diff
@@ -103,22 +112,31 @@ func (d *Device) ToClient(id int, msg interface{}) error {
 	return client.Send(msg)
 }
 
+// preDevice holds information about a device between pre-registration and registration.
```

Comment:
nit: would the name `preRegProcess` be more appropriate here? the `preDevice` name is descriptive by itself.